src/engine/reflection.h

```diff
@@ -13,7 +13,9 @@
 #define LUMIX_FUNC(F) function<&F>(#F, #F)
 #define LUMIX_CMP(Cmp, Name, Label) cmp<&ReflScene::create##Cmp, &ReflScene::destroy##Cmp>(Name, Label)
 #define LUMIX_PROP(Property, Label) prop<&ReflScene::get##Property, &ReflScene::set##Property>(Label)
+#define LUMIX_READONLY_PROP(Property, Label) prop<&ReflScene::get##Property>(Label)
```

Comment:
this macro (and the next one) is not necessary, since it does not shorten or duplicate anything

Reply:
How will you prevent the error for SetXXX not existing?

Reply:
```
template <auto Getter, auto Setter = nullptr>
	builder& enum_prop(const char* name) {
```